js/plugins/ollama/src/index.ts

```diff
@@ -279,8 +281,18 @@ function readChunks(reader) {
   };
 }
 
-function getPrompt(input: GenerateRequest) {
-  return input.messages.map((m) => m.content.map((c) => c.text).join()).join();
+function getPrompt(input: GenerateRequest): string {
+  return input.messages
+    .filter((m) => m.role !== 'system') // Exclude system messages
+    .map((m) => m.content.map((c) => c.text).join())
+    .join();
+}
+
+function getSystemMessage(input: GenerateRequest): string {
+  return input.messages
+    .filter((m) => m.role === 'system') // Exclude system messages
```

Comment:
Can you just update the comment here

i.e.) `// Include only the system messages`

Or alternatively, just drop it entirely. :)